app/server/tests/api-login.test.js

```diff
@@ -2,7 +2,7 @@ const { AuthFunctions } = require('./functions/authFunctions')
 
 let users = new AuthFunctions(); // build class for user actions
 
-describe('Testing optimal inputs for login', () => {
+describe('Testing inputs that are expected to succeed login', () => {
```

Comment:
Should we outline in a comment above what the constraints actually are?

Reply:
Added some comments.